src/components/App/App.js

```diff
@@ -0,0 +1 @@
+App.js
```

Comment:
Codacy found an issue: ['App' is not defined.](https://app.codacy.com/gh/WebHeroSchool/evafinal/pullRequest?prid=5874309)